frontend/app/src/store/balances/blockchain-accounts.ts

```diff
@@ -1113,14 +1127,16 @@ export const useBlockchainAccountsStore = defineStore(
             true
           );
 
-          await fetchDetectedTokens();
+          await fetchDetectedTokens(null);
```

Comment:
Wouldn't it make more sense to make the default `null` or make it optional to avoid calling it null everywhere?